schemas/shacl/valid.jsonld

```diff
@@ -0,0 +1,44 @@
+{
+  "@context": {
+    "@vocab": "https://schema.org/",
```

Comment:
Or is it clearer to use a prefix for Schema too?

```suggestion
    "schema": "https://schema.org/",
```

Reply:
The `codemeta.jsonld` we typically include (our harvester does) has an elaborate `@context` that maps all the specific terms so the json the developers provide can be kept fairly simple. Perhaps for your more minimal example it might make sense to use a prefix for schema.